[2024-02-21T07:38:07.545+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-03T00:00:00+00:00 [queued]>
[2024-02-21T07:38:07.560+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-03T00:00:00+00:00 [queued]>
[2024-02-21T07:38:07.560+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:38:07.578+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-03 00:00:00+00:00
[2024-02-21T07:38:07.588+0000] {standard_task_runner.py:60} INFO - Started process 108 to run task
[2024-02-21T07:38:07.597+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-03T00:00:00+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpf092wxvu']
[2024-02-21T07:38:07.600+0000] {standard_task_runner.py:88} INFO - Job 74: Subtask get_mail
[2024-02-21T07:38:07.699+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-03T00:00:00+00:00 [running]> on host 893865381b0d
[2024-02-21T07:38:07.846+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-03T00:00:00+00:00'
[2024-02-21T07:38:07.847+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 196, in execute
    context_merge(context, self.op_kwargs, templates_dict=self.templates_dict)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/utils/context.py", line 261, in context_merge
    context.update(*args, **kwargs)
  File "/usr/local/lib/python3.8/_collections_abc.py", line 837, in update
    for key, value in other:
ValueError: too many values to unpack (expected 2)
[2024-02-21T07:38:07.868+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=extract_email, task_id=get_mail, execution_date=20240203T000000, start_date=20240221T073807, end_date=20240221T073807
[2024-02-21T07:38:07.881+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 74 for task get_mail (too many values to unpack (expected 2); 108)
[2024-02-21T07:38:07.908+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-21T07:38:08.029+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-02-21T07:59:38.966+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-03T00:00:00+00:00 [queued]>
[2024-02-21T07:59:38.975+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_email.get_mail scheduled__2024-02-03T00:00:00+00:00 [queued]>
[2024-02-21T07:59:38.975+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-02-21T07:59:38.986+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): get_mail> on 2024-02-03 00:00:00+00:00
[2024-02-21T07:59:38.991+0000] {standard_task_runner.py:60} INFO - Started process 372 to run task
[2024-02-21T07:59:38.994+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'extract_email', 'get_mail', 'scheduled__2024-02-03T00:00:00+00:00', '--job-id', '129', '--raw', '--subdir', 'DAGS_FOLDER/extract_email.py', '--cfg-path', '/tmp/tmpg0ce7n2i']
[2024-02-21T07:59:38.996+0000] {standard_task_runner.py:88} INFO - Job 129: Subtask get_mail
[2024-02-21T07:59:39.033+0000] {task_command.py:423} INFO - Running <TaskInstance: extract_email.get_mail scheduled__2024-02-03T00:00:00+00:00 [running]> on host 9bab6a0db245
[2024-02-21T07:59:39.106+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_email' AIRFLOW_CTX_TASK_ID='get_mail' AIRFLOW_CTX_EXECUTION_DATE='2024-02-03T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-03T00:00:00+00:00'
[2024-02-21T07:59:42.277+0000] {ImapSmtp.py:839} INFO - IMAP search: 'gmail:'Message from RPA Python' after:15-12-2023'
[2024-02-21T07:59:45.486+0000] {logging_mixin.py:188} INFO - Message from RPA Python0
[2024-02-21T07:59:45.487+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-162.txt')
[2024-02-21T07:59:45.487+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.487+0000] {logging_mixin.py:188} INFO - Message from RPA Python1
[2024-02-21T07:59:45.488+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-163.txt')
[2024-02-21T07:59:45.488+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.489+0000] {logging_mixin.py:188} INFO - Message from RPA Python2
[2024-02-21T07:59:45.489+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-164.txt')
[2024-02-21T07:59:45.490+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.490+0000] {logging_mixin.py:188} INFO - Message from RPA Python3
[2024-02-21T07:59:45.491+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-165.txt')
[2024-02-21T07:59:45.491+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.491+0000] {logging_mixin.py:188} INFO - Message from RPA Python4
[2024-02-21T07:59:45.493+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-166.txt')
[2024-02-21T07:59:45.493+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.493+0000] {logging_mixin.py:188} INFO - Message from RPA Python5
[2024-02-21T07:59:45.494+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-167.txt')
[2024-02-21T07:59:45.494+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.495+0000] {logging_mixin.py:188} INFO - Message from RPA Python6
[2024-02-21T07:59:45.496+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-168.txt')
[2024-02-21T07:59:45.496+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.496+0000] {logging_mixin.py:188} INFO - Message from RPA Python7
[2024-02-21T07:59:45.497+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-169.txt')
[2024-02-21T07:59:45.497+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.498+0000] {logging_mixin.py:188} INFO - Message from RPA Python8
[2024-02-21T07:59:45.498+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-170.txt')
[2024-02-21T07:59:45.499+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:45.499+0000] {logging_mixin.py:188} INFO - Message from RPA Python9
[2024-02-21T07:59:45.500+0000] {ImapSmtp.py:1240} INFO - Attachment filepath: PosixPath('/data/attachments/test-171.txt')
[2024-02-21T07:59:45.501+0000] {ImapSmtp.py:1243} INFO - Saving attachment: test.txt
[2024-02-21T07:59:46.125+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-21T07:59:46.161+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=extract_email, task_id=get_mail, execution_date=20240203T000000, start_date=20240221T075938, end_date=20240221T075946
[2024-02-21T07:59:46.267+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-21T07:59:46.284+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
